interop/xds/custom_lb_test.go

```diff
@@ -64,14 +64,14 @@ func (s) TestCustomLB(t *testing.T) {
 				errCh.Send(errors.New("failed to receive metadata"))
 				return &testpb.SimpleResponse{}, nil
 			}
-			rpcBMD := md.Get(RPCBehaviorMetadataKey)
+			rpcBMD := md.Get("rpc-behavior")
 			if len(rpcBMD) != 1 {
-				errCh.Send(errors.New("only one value should be received for metadata key " + RPCBehaviorMetadataKey))
+				errCh.Send(errors.New("only one value should be received for metadata key " + "rpc-behavior"))
```

Comment:
No need for concatenation here, it's one string. "...metadata key: rpc-behavior"